Forms.php

```diff
@@ -606,6 +606,10 @@ protected static function _input_gen( $args ) {
 	 * @return string
 	 */
 	protected static function add_label( $input, $desc, $desc_pos ) {
+		if ( empty( $desc ) ) {
```

Comment:
I think it would be better if you checked "$desc === null", so that there's a difference between no description passed and a description that's simply an empty string.